spec/features/course/assessment/submissions_viewing_spec.rb

```diff
@@ -36,6 +37,36 @@
           )
         end
       end
+
+      scenario 'I can view pending submissions' do
+        students = create_list(:course_student, 4, :approved, course: course)
+        attempting_submission, submitted_submission1, submitted_submission2, graded_submission =
+          students.zip([:attempting, :submitted, :submitted, :graded]).map do |student, trait|
+            create(:course_assessment_submission, trait,
+                   assessment: assessment, course: course,
+                   creator: student.user, course_user: student)
+          end
+
+        # Staff without group can view all pending submissions
+        visit pending_course_submissions_path(course)
+        save_page
+        expect(page).to have_content_tag_for(submitted_submission1)
+        expect(page).to have_content_tag_for(submitted_submission2)
+        expect(page).not_to have_content_tag_for(attempting_submission)
+        expect(page).not_to have_content_tag_for(graded_submission)
+
+        # Staff with group view pending submissions of own group students
+        group = create(:course_group, course: course)
+        create(:course_group_manager, group: group, course: course, course_user: course_manager)
+        create(:course_group_user, group: group, course: course,
+                                   course_user: submitted_submission1.course_user)
```

Comment:
For now, yes. This has got something to do with the factory, because there are multiple possible configurations (group and course_user, group, course_user and course, etc). Will fix it in another PR.